packages/core/src/embeddings/HuggingFaceEmbedding.ts

```diff
@@ -46,3 +47,64 @@ export class HuggingFaceEmbedding extends BaseEmbedding {
     return Array.from(output.data);
   }
 }
+
+// TODO workaround issue with @huggingface/inference@2.7.0
```

Comment:
maybe move this to `HuggingFaceInferenceAPIEmbedding`? as it's very different to the inference via transformers.js?